Makefile

```diff
@@ -28,7 +31,7 @@ config-git:
 subctl: config-git
 	./scripts/subctl.sh $(SUBCTL_ARGS)
 
-e2e: $(if $(TARGET_RELEASE),deploy)
+e2e: $(if $(DEPLOY_NEEDED),deploy)
```

Comment:
Why doesn’t `e2e` need `deploy` when we’re branching? Isn’t the reality that we shouldn’t need `e2e` when we’re branching?